.storybook/public/mockServiceWorker.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
```

Comment:
Have you tried to use the official https://github.com/mswjs/msw-storybook-addon?

Reply:
i tried it out, but it seemed more complicated to use and not lot of docs around it yet, so I chose to remain as is

Reply:
> added [msw library](https://github.com/mswjs/msw) to be able to create storybooks for the various states for DownloadScript that used suspense and error boundary. I can't figure out why, but I would occasionally get 404 error messages when viewing storybook using msw, but it is resolved by refreshing the browser... other than that it works great

Maybe this would solve this problem?

Reply:
when i tried it out, it didn't. I think it has something to do with hot reload, but i'm not sure how to reliably trigger it.